scripts/workers/pr-care-worker-run.sh

```
@@ -0,0 +1,74 @@
+#!/bin/bash
+# pr-care-run.sh — 단발성. stale PR 케어.
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_DIR="/Users/mark/workspace/minglit"
+PROMPT_FILE="$SCRIPT_DIR/prompts/pr-care-worker.txt"
```

Shellcheck (0.11.0)

[warning] PROMPT_FILE appears unused. Verify use (or export if used externally).

(SC2034)